packages/yew/src/dom_bundle/traits.rs

```diff
@@ -0,0 +1,88 @@
+use super::BNode;
+use crate::html::AnyScope;
+use crate::html::NodeRef;
+use web_sys::Element;
+
+pub(super) trait ReconcileTarget {
```

Comment:
A little doc comment explaining what it's for and why it's needed would be nice here